### DISCUSSION TITLE
MultiSystems cannot generate deepmd

### DISCUSSION THREAD

**Discussion by warriors1523** [deepmodeling/dpdata#263] [Q&A]
![2](https://user-images.githubusercontent.com/92622311/158624383-dda46235-1ffe-4e2b-a23a-30617b917a97.png)
![1](https://user-images.githubusercontent.com/92622311/158624414-51fc3bcb-bc6c-409f-8694-4da9fc669ecc.png)
![3](https://user-images.githubusercontent.com/92622311/158624695-58c9e6ac-5cb9-4043-ac05-136ba4a6d461.png)

**Comment by wanghan-iapcm** [comment]
See if #248 is the same issue

**Comment by hu-lm** [comment]
Hi, I have the same problem. Did you solve that? Thanks!
